ENIGMAsystem/SHELL/Universal_System/Extensions/ttf/ttf.cpp

```diff
@@ -37,32 +45,134 @@ class FontManager {
   static FT_Library library;
 };
 
+struct GlyphPair {
+  GlyphPair(unsigned index = 0, unsigned area = 0) : index(index), area(area) {}
+  unsigned index;
+  unsigned area;
+};
+
 FT_Library FontManager::library;
 static bool FreeTypeAlive = FontManager::Init();
 
+std::vector<std::string> fontSearchPaths;
+
+// TODO: Fill in this map in
+// https://docs.microsoft.com/en-us/typography/fonts/windows_10_font_list
+const std::map<std::string, std::string> msFontsNames = {
```

Comment:
@RobertBColton it is possible to generate this map on windows using win32 crap but we would want access to it on linux and os x as well as it is possible to install msfonts on linux. I'd appreciate it if you could fill out that map cause I cba. I linked the reference above but you could also write a 4 line app to generate that map too